Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -70,6 +70,11 @@ catch
    Write-Verbose "Unable to get the authScheme $error" 
 }
 
+if ($endpoint.Auth.parameters.authenticationType -eq "spnCertificate")
```

Comment:
ensure there will be no null exception for this if condition

Reply:
verified.